src/components/Modal_Components/ManualTransaction.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
```

Comment:
We can remove the useEffect import here if it was just used for a dev console log